proprietary/Components/src/denhaag/anchor-navigation.tokens.json

```diff
@@ -0,0 +1,22 @@
+{
+  "denhaag": {
+    "anchor-navigation": {
+      "border-inline-start": { "value": "2px solid {denhaag.color.grey.2}" },
+      "ul": {
+        "list-style": { "value": "none" },
+        "padding-inline-start": { "value": "1.125rem" }
+      },
+      "anchor": {
+        "background-color": { "value": "{denhaag.color.green.3}" },
+        "color-default": { "value": "{denhaag.color.grey.4}" },
```

Comment:
1: `color` should be enough instead of `color-default`
2: By defining the `color-interactive` this way you make a design choice, therefor i suggest a hover, focus & visited state in the tokens.